perf/istio/external-traffic/templates/sidecar.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Sidecar
+metadata:
+  namespace: allow-external-traffic-a
+  name: allow-external-traffic-a-sidecar
+spec:
+  egress:
+    - hosts:
+        - allow-external-traffic-a/*
```

Comment:
Isn’t there a way to say self/* so as to not name the namespace ? @rshriram

Reply:
Looks like `.` refers to "self". 

https://github.com/istio/istio/blob/727e719b56362060924cd75bef6ed731cc41b272/pilot/pkg/model/validation.go#L633

Reply:
I think even better - `~` means import nothing. I believe this will give us the behavior we want. If we do `./*` ports can still be blocked if we define service entries in the namespace. With `~/*`, we will be able to access any port regardless (because `global.outboundTrafficPolicy.mode=ALLOW_ANY` is set)